tapgarden/planter.go

```diff
@@ -135,6 +135,27 @@ type FinalizeParams struct {
 	SiblingTapTree fn.Option[asset.TapscriptTreeNodes]
 }
 
+// FundParams are the options available to change how a batch is funded, and how
+// the genesis TX is constructed.
+type FundParams struct {
+	FeeRate        fn.Option[chainfee.SatPerKWeight]
+	SiblingTapTree fn.Option[asset.TapscriptTreeNodes]
```

Comment:
Comment above says a genesis point, but looks like this is mainly about proving the sibling tap tree? Also thought we already had that plumbing hooked up here.